### DISCUSSION TITLE
Disable connection on top and bottom element

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1894] [Q&A] [answered]
Hello everyone,

I would like to know if it is possible to disable the connection or force the connection to the left or right.

Here are the screenshots: The first two screenshots are the action I don't want to do when I move the item, I would like it to connect automatically to the sides.

<img width="182" alt="Capture d’écran 2022-11-15 112556" src="https://user-images.githubusercontent.com/83480447/201896895-df5124cc-7b7f-4247-a326-85ab1d124d76.png">
<img width="227" alt="Capture d’écran 2022-11-15 112613" src="https://user-images.githubusercontent.com/83480447/201896906-6142184e-a5d8-44b1-b8c0-33d715c9b48d.png">

**Comment by kumilingus** [comment, marked as answer]
What link `router` do you use?

> Manhattan

---

You can specify the `startDirections` and `endDirections` . Please refer to [docs](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#routers.manhattan).

Here's an [example](https://jsfiddle.net/kumilingus/w0rkposx/) of how to set the directions dynamically.

**Reply by malomiquel** [reply]
I use boundary router

**Reply by kumilingus** [reply]
`boundary` is a `connectionPoint`. `router` will be perhaps the `orthogonal` or `manhattan`?

**Reply by malomiquel** [reply]
Ah yes I use Manhattan

**Reply by kumilingus** [reply]
I've updated the answer.
